client/my-sites/github-deployments/components/repositories/repository-list.tsx

```diff
@@ -71,42 +63,6 @@ export const GitHubBrowseRepositoriesList = ( {
 		return <NoResults manageInstallationUrl={ installation.management_url } />;
 	}
 
-	const openPermissions: MouseEventHandler = ( event ) => {
-		event.preventDefault();
-
-		const openedPopup = openPopup( {
-			url: installation.management_url,
-			target: '_blank',
-			onMessage: ( data, popup ) => {
```

Comment:
This will never work. See: p1710976659813219/1710805745.526269-slack-C06D9M3CHMK